src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.dao.UserDaoImpl;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    BetDao betDao = new BetDaoImpl();
+    UserDao userDao = new UserDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] betData = command.split(" ");
+                String name = betData[0];
+                String lastName = betData[1];
+                int value = Integer.parseInt(betData[2]);
+                double risk = Double.parseDouble(betData[3]);
+                bet = new Bet(value, risk);
+                user = new User(name, lastName);
+            } catch (NumberFormatException e) {
```

Comment:
is there a risk of ArrayIndexOutOfBoundsException?

Reply:
done